### PR TITLE
Updated the Service Map based off latest info

### PR DESCRIPTION
**Pull request by mbcrump**
Updated the Service Map to match the new interactive PDF recently created. Includes text tweaks, new table header, most recent service mapping information.

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mbcrump/architecture-center/blob/patch-1/docs/aws-professional/services.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/aws-professional/services?branch=pr-en-us-240) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C29%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201709291850330155-240%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YOgPMF7ytgwDuY%2BG%2FUeOdZzb0wRB4049kce78sxiBwM%3D&st=2017-09-29T18%3A48%3A06Z&se=2017-09-30T18%3A58%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/aws-professional/services.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ ms.date: 03/24/2017
...
+# Azure & AWS Cloud Service Map
 
-This article helps you understand how Microsoft Azure services compare to Amazon Web Services (AWS). Whether you are planning a multicloud solution with Azure and AWS, or migrating to Azure, you can compare the IT capabilities of Azure and AWS services in all categories.
+This document defines how common cloud services are made available via Azure and Amazon Web Services (AWS). We hope that this guide helps developers understand the services that Azure offers, whether they are new to the cloud or just new to Azure. We’ve structured the guide using a table that explains each cloud service capability sorted by service popularity, and maps the capability to the service offering name for both Azure and AWS. In the following tables, there are multiple Azure services listed for some AWS services. The Azure services are similar to one another, but depth and breadth of capabilities vary.
 
-In the following tables, there are multiple Azure services listed for some AWS services. The Azure services are similar to one another, but depth and breadth of capabilities vary.
+[Downloada PDF of the Azure & AWS Cloud Service Map](https://aka.ms/awsazureguide)
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: Right now this link goes to `https://www.microsoft.com/en-us/`

File: docs/aws-professional/services.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ ms.date: 03/24/2017
...
 
-## Azure and AWS for multicloud solutions
+## Azure and AWS for Multicloud Solutions
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: Original is correct. Microsoft docs style for headings is sentence caps, not title caps.

File: docs/aws-professional/services.md (hunk 1 above)
Comment (MikeWasson) [reply docs/aws-professional/services.md]: need space between "download a"

File: docs/aws-professional/services.md [hunk 3]
```diff
@@ -7,13 +7,13 @@ ms.date: 03/24/2017
 
 pnp.series.title: Azure for AWS Professionals
 ---
-# AWS to Azure services comparison
+# Azure & AWS Cloud Service Map
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: imo the original title is more clear

File: docs/aws-professional/services.md [hunk 4]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
-| Marketplace | AWS Marketplace | [Azure Marketplace](https://azure.microsoft.com/marketplace/) | Easy-to-deploy and automatically configured third-party applications, including single virtual machine or multiple virtual machine solutions. |
+| Area        | AWS service     | Azure service                                                 | Description                                                                                                                                               |
+|-------------|-----------------|---------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Marketplace | AWS Marketplace | [Azure Marketplace](https://azure.microsoft.com/marketplace/) | Easy-to-deploy and automatically configure third- AWS Marketplace Marketplace party applications, including single or multiple virtual machine solutions. |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: Something garbled here? ("third- AWS Marketplace Marketplace party applications")

File: docs/aws-professional/services.md [hunk 5]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
+| Service                            | AWS                     | Azure                                                                                                                                                                                                                                                                                                                                                                | Description                                                                                                                                                                                                                                     |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: This column is not really a list of "services" per se - e.g., "backend process orchestration" is not a service. I think that's why the original column heading says "Area"

File: docs/aws-professional/services.md [hunk 6]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
+| **&nbsp;**                      | Amazon Lightsail                | [Virtual Machines & Images](https://azure.microsoft.com/services/virtual-machines/)                                                                                                                                                                                                                                                                              | Offers a lightweight, simplified product offering users can choose from from when building out a virtual machine.                                                                                                                                               |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: Not sure what "lightweight" means in the context

File: docs/aws-professional/services.md [hunk 7]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
+| Domain name system management | Route 53               | [DNS](https://azure.microsoft.com/services/dns/)                                                                                                      | Manage DNS records using the same credentials, billing, and support contract as other Azure services.                                                                                |
+| **&nbsp;**                    | Route 53               | [Traffic Manager](https://azure.microsoft.com/services/traffic-manager/)                                                                                    | Service that hosts domain names, routes users to Internet applications, manages traffic to apps, and improves app availability with automatic failover.                                                |
+| Content delivery network      | CloudFront             | [Content Delivery Network](https://azure.microsoft.com/services/cdn/)                                                                                 | Global content delivery network that transfers audio, video, applications, images, and other files.                           |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: "Transfer" doesn't seem to be the right term here. Serves or delivers. 

(I guess "delivers" was slightly redundant but here, but I think it's OK.)

File: docs/aws-professional/services.md [hunk 8]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
 |-------------------------|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Relational database     | RDS                                  | [SQL Database](https://azure.microsoft.com/services/sql-database/)                                       | Relational database-as-a-service (DBaaS) where the database resilience, scale, and maintenance are primarily handled by the platform.                   |
+| Relational database     | RDS                                  | [SQL Database including Postgres and MySQL](https://azure.microsoft.com/services/sql-database/)                                       | SQL Database is a high-performance, reliable, and secure database you can use to build data-driven applications and websites, without needing to manage infrastructure.                 |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: Provide separate links for Postgres and MySQL?

File: docs/aws-professional/services.md [hunk 9]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
+| **&nbsp;**         | AWS IoT                                                                                 | [IoT Hub](https://azure.microsoft.com/services/iot-hub/)                                                                             | A cloud gateway for managing bidirectional communication with billions of IoT devices, securely and at scale.                                                     |
+| **&nbsp;**         | AWS Greengrass - Software for Connected Devices                                         | [IoT Edge](https://azure.microsoft.com/services/iot-hub/iot-gateway-sdk/)                                                     | Contains the infrastructure and modules to create IoT gateway solutions.  |
+| Streaming data     | Kinesis Firehose <br/><br/>Kinesis Streams                                                        | [Event Hubs](https://azure.microsoft.com/services/event-hubs/)                                                                             | Allow mass ingestion of small data inputs, typically from devices and sensors, to process and route data.                                 |
 
 
-## Management & monitoring
+## Management & Monitoring
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: original is correct. Microsoft docs style for heading is sentence casing

File: docs/aws-professional/services.md [hunk 10]
```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
-| Cloud advisor                     | Trusted Advisor                   | [Azure Advisor](https://azure.microsoft.com/services/advisor/)                                                                                                                                                                                                                                                                                      | Provides analysis of cloud resource configuration and security so subscribers can ensure they’re making use of best practices and optimum configurations.                                                                                 |
-| Deployment orchestration (DevOps) | OpsWorks (Chef-based)             | [Azure Automation](https://azure.microsoft.com/services/automation/)                                                                                                                                                                                                                | Con
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: browserbased  => browser-based

Or "An authenticated shell experience that runs in a browser" ?

Comment (MikeWasson) [reply docs/aws-professional/services.md]: Add "(Preview)"?  

Not sure how consistent we are in this doc about indicating preview services, but we do include it in some entries.

Same comment for Manages Service Identity and Event Grid

Comment (MikeWasson) [inline docs/aws-professional/services.md]: Active Directory != Azure Active Directory

Spell out "Azure Active Directory"?

Comment (MikeWasson) [inline docs/aws-professional/services.md]: Two while clauses in one sentence is awkward, can we re-word this or revert to two sentences?

File: docs/aws-professional/services.md [hunk 11]
```diff
@@ -192,23 +192,23 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
...
-| Content management in the cloud      | None                            | [SharePoint Online](https://products.office.com/sharepoint) | Provides a collaborative way for individuals, teams, and organizations to intelligently discover, share, and collaborate on content from anywhere and on any device.                           |
-| Commercial PAAS-IAAS-DBaaS framework | None                            | [Azure Stack](https://azure.microsoft.com/overview/azure-stack/)                                    | A hybrid cloud platform that lets you deliver Azure services from your organization’s datacenter.                                                                                              |
+| Enterprise app integration           |   None                          | [Logic Apps](https://azure.microsoft.com/services/logic-apps/)                    | Connects apps, data, and devices anywhere with cloudbased connectors that includes Salesforce, Office 365, Twitter, Dropbox, Google services, and more.                                                     |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: cloudbased  => cloud-based

**Comment by SnapAnalytxInc** [comment]
Thanks Mike..   @mbcrump this is a lot of changes and issues in composition.  I'll have time next week to go and review.

File: docs/aws-professional/services.md (hunk 11 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: agreed - it should be cloud-based

File: docs/aws-professional/services.md (hunk 10 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: Sure. I don't see the while you mentioned or I'd help provide a suggestion.

File: docs/aws-professional/services.md (hunk 9 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: OK

File: docs/aws-professional/services.md (hunk 8 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: I think separate links may help here. The key is that folks don't know we have support for those additional two dbs.

File: docs/aws-professional/services.md (hunk 7 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: delivers if fine

File: docs/aws-professional/services.md (hunk 6 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: lightweight is referred to in this context as "you pick the software" that you want installed. It also has been noted that "community-led templates" isn't accurate for Lightsail.

File: docs/aws-professional/services.md (hunk 4 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: Yes. the corrected text is "Easy-to-deploy and automatically configure third-party applications, including single or multiple"
virtual machine solutions.

File: docs/aws-professional/services.md (hunk 5 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: I find "service" appears to be a better description, but we can use either.

File: docs/aws-professional/services.md (hunk 3 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: I see this as more of a "mapping" between service vs. a comparison. If it was a comparison, I'd expect AWS does x and Azure does y.

File: docs/aws-professional/services.md (hunk 2 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: OK

File: docs/aws-professional/services.md (hunk 1 above)
Comment (mbcrump) [reply docs/aws-professional/services.md]: The PDF was just updated on MS Download. It works properly now.

**Comment by lbrader** [comment]
@mbcrump This PR changes far more then expected.  Let's close this PR.  Create a new PR and only update descriptions, or add new products -thanks.

**Pull request closed without merging**